src/sqlalchemy_oso_cloud/orm.py

```diff
@@ -89,4 +99,42 @@ def remote_relation(remote_resource_name: str, remote_relation_key: Optional[str
   """
   col = mapped_column(*args, **kwargs)
   col.column.info[_REMOTE_RELATION_INFO_KEY] = (remote_resource_name, remote_relation_key)
+  return col
+
+def actor(*args, actor_type: Optional[str] = None, **kwargs) -> MappedColumn:
```

Comment:
thought: I wonder if it would make sense for these functions to be class methods on the `Role[Mapping]` class, to make it clearer what they're doing. E.g. `agent_id = Role.actor(String(50), ...)`. A function named `actor` on its own seems a bit general/ambiguous to a reader who doesn't know it's connected to the `Role[Mapping]` mixin